packages/blade/src/utils/getColorScheme/getColorScheme.web.test.ts

```diff
@@ -1,5 +1,5 @@
 import MatchMediaMock from 'jest-matchmedia-mock';
-import getColorScheme from '../../getColorScheme';
+import getColorScheme from './';
```

Comment:
Yes, this looks clearer for understanding. Is there any other way to pick web and native files? For example how metro can pick `.android` or `.ios` file even if we import the file by it's name directly.

Reply:
yes `import getColorScheme from '../getColorScheme';` this will work and will pickup the name correctly but what I want to test here is that the re-exports are working correctly from `index` else there's no clear way for me to test if my re-exports work perfectly